assets/index.js

```diff
@@ -10,7 +10,6 @@ const
 	SEARCH_STATE_KEY = 'APEXDOX_SEARCH_RESULTS';
 
 const highlightJsSelectors = [
```

Comment:
@dschach does this need to be updated?

Reply:
This block is updated because the JS library requires calling it in new ways. I renamed the variable because these blocks are being specified as Apex. If this extension ever grows to cover LWC or other metadata, we could cover other languages. Or if an org uses functions, we could cover those languages. Apex is just specific here. If I'm way off, I hear you.

Reply:
Sorry, I'm ok with the rename - what I mean here was, I think this is a bug, as this line has actually not been updated (I can see how the github suggestion feature is a little confusing, though, because it actually shows the diff of what we want). Within `initHighlightJs` a variable named `apexJsSelectors` is referenced and operated on, however the initialization of that variable does not exist so far as I can tell. This line needs to be updated to the new name.